Assets/HoloToolkit-Examples/InteractiveElements/Scripts/Controls/ButtonThemeWidget.cs

```diff
@@ -30,6 +31,9 @@ public class ButtonThemeWidget : InteractiveThemeWidget
         [Tooltip("scale animation component: optional")]
         public ScaleToValue ScaleSize;
 
+        [Tooltip("Upate the widget's theme and refresh if the theme tag changes")]
+        public bool WatchForThemeChange = false;
```

Comment:
I will remove the check for WatchForThemeChange and just make it automatic for now. If anyone feels there's a potential break, let me know.